docs/third-party-packages.md

```diff
@@ -46,6 +46,14 @@ Checkout <a href="https://github.com/taoufik07/nejma-chat" target="_blank">nejma
 Another solution for websocket broadcast. Send messages to channel groups from any part of your code.
 Checkout <a href="https://channel-box.andrey-sobolev.ru/" target="_blank">MySimpleChat</a>, a simple chat application built using `channel-box` and `starlette`.
 
+### Starlette+RabbitMQ
+
+<a href="https://github.com/Sobolev5/Starlette-plus-RabbitMQ" target="_blank">GitHub</a>
+
+Starlette + RabbitMQ. Simple RPC calls to backend microservices. 
+Checkout <a href="http://31.210.172.230:5888/" target="_blank">Full working example</a>.
```

Comment:
Wow! @Sobolev5 Thank you for demo. What do you think about using FQDN instead IP for it?

Reply:
Good afternoon. The main purpose of this demo is simplicity. So that everyone can run this example and it works for him without too much effort.
Therefore, I specifically did not specify the domain, although it would be quite easy to do (proxying through Nginx)

Reply:
I make a FQDN http://starlette-plus-rabbitmq.andrey-sobolev.ru/
If it is necessary, I will correct the commit.